src/main.rs

```diff
@@ -247,6 +250,18 @@ fn parse_hosts_line(line: &str) -> Option<String> {
     None
 }
 
+fn parse_dnsmasq_line(line: &str) -> Option<String> {
+    lazy_static! {
+        static ref RE: Regex = Regex::new(r#"(?:address|server)=/(.*)/.*"#).unwrap();
+    }
+
+    for cap in RE.captures_iter(line) {
+        return Some(String::from(&cap[1]));
+    }
+
+    None
```

Comment:
This for-loop is an error in clippy (`clippy::never_loop`), because it doesn't actually loop. It would be clearer to rewrite this section as:
```suggestion
let cap = RE.captures(line)?;
Some(String::from(&cap[1]))
```

Reply:
good catch!